web/components/nav/sidebar.tsx

```diff
@@ -108,122 +95,77 @@ const logout = async () => {
+    // loggedIn && { name: 'Get M$', href: '/add-funds', icon: CashIcon },
     { name: 'Groups', href: '/groups' },
-    { name: 'Refer a friend', href: '/referrals' },
+    loggedIn && { name: 'Refer a friend', href: '/referrals' },
     { name: 'Charity', href: '/charity' },
     { name: 'Labs', href: '/labs' },
-    { name: 'Discord', href: 'https://discord.gg/eHQBNBqXuh' },
-    { name: 'Help & About', href: 'https://help.manifold.markets/' },
-    {
-      name: 'Sign out',
-      href: '#',
-      onClick: logout,
-    }
+    { name: 'Blog', href: 'https://news.manifold.markets' },
+    { name: 'Discord', href: 'https://discord.gg/eHQBNBqXuh' }
+    // { name: 'Twitter', href: 'https://twitter.com/ManifoldMarkets' },
   )
 }
 
-const signedOutDesktopNavigation = [
-  { name: 'Home', href: '/', icon: HomeIcon },
-  { name: 'Explore', href: '/search', icon: SearchIcon },
-  { name: 'Midterms', href: '/midterms', icon: FlagIcon },
-  {
-    name: 'Help & About',
-    href: 'https://help.manifold.markets/',
-    icon: BookOpenIcon,
-  },
-]
-
-const signedOutMobileNavigation = [
-  {
-    name: 'Help & About',
-    href: 'https://help.manifold.markets/',
-    icon: BookOpenIcon,
-  },
-  { name: 'Midterms', href: '/midterms', icon: FlagIcon },
-  { name: 'Charity', href: '/charity', icon: HeartIcon },
-  { name: 'Tournaments', href: '/tournaments', icon: TrophyIcon },
-  { name: 'Leaderboards', href: '/leaderboards', icon: ChartBarIcon },
-  { name: 'Discord', href: 'https://discord.gg/eHQBNBqXuh', icon: ChatIcon },
-]
-
-const signedInMobileNavigation = buildArray(
-  { name: 'Search', href: '/search', icon: SearchIcon },
-
-  !IS_PRIVATE_MANIFOLD && [
+const getMobileNav = (loggedIn: boolean) => {
+  if (IS_PRIVATE_MANIFOLD) {
+    return [{ name: 'Leaderboards', href: '/leaderboards', icon: ChartBarIcon }]
+  }
+  return buildArray(
     { name: 'Midterms', href: '/midterms', icon: FlagIcon },
     { name: 'Tournaments', href: '/tournaments', icon: TrophyIcon },
     { name: 'Leaderboards', href: '/leaderboards', icon: ChartBarIcon },
-    { name: 'Get M$', href: '/add-funds', icon: CashIcon },
-  ],
-  {
-    name: 'Help & About',
-    href: 'https://help.manifold.markets/',
-    icon: BookOpenIcon,
-  }
-)
-
-function getMoreMobileNav() {
-  const signOut = {
-    name: 'Sign out',
-    href: '#',
-    onClick: logout,
-  }
-  if (IS_PRIVATE_MANIFOLD) return [signOut]
-
-  return buildArray<MenuItem>(
-    { name: 'Groups', href: '/groups' },
-    { name: 'Refer a friend', href: '/referrals' },
-    { name: 'Charity', href: '/charity' },
-    { name: 'Labs', href: '/labs' },
-    { name: 'Discord', href: 'https://discord.gg/eHQBNBqXuh' },
-    signOut
+    loggedIn && { name: 'Groups', href: '/groups', icon: UsersIcon },
+    // { name: 'Charity', href: '/charity', icon: HeartIcon },
+    // loggedIn && { name: 'Get M$', href: '/add-funds', icon: CashIcon },
+    { name: 'Labs', href: '/labs', icon: BeakerIcon }
```

Comment:
I think Refer a friend is important to keep. I've heard from users that referred other users by finding this exact option in the sub menu.

If anything we should put this option in more places. Not take it away, even just for mobile.